macos/FluentUI/Separator.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+@objc(MSFSeparatorStyle)
+public enum SeparatorStyle: Int {
+    case `default`
+    case shadow
+
+    fileprivate var color: NSColor {
+		switch self {
+		case .default:
+			if #available(OSX 10.14, *) {
+				return NSColor.separatorColor
+
+			} else {
+				return NSColor.systemGray
```

Comment:
[NSColor's quaternaryLabelColor](https://developer.apple.com/documentation/appkit/nscolor/1534635-quaternarylabelcolor) might be a better fallback? "Use this color for the least important text in your interface and for separators between text items."

Whereas [separatorColor](https://developer.apple.com/documentation/appkit/nscolor/2998831-separatorcolor) is "The color to use for separators between different sections of content. Do not use this color for split view dividers or window chrome dividers."

The distinction between those two types of separators is maybe interesting on macOS.

Reply:
Or if we can't get a solid read on what colors we want as a fallback, maybe just defining the fallback color in the asset catalog would work.

Reply:
Forgot that .gridColor is probably the best fallback, and also what design requested. Switched to that.